tests/sampling/test_jax.py

```diff
@@ -333,20 +333,26 @@ def test_get_batched_jittered_initial_points():
     with pm.Model() as model:
         x = pm.MvNormal("x", mu=np.zeros(3), cov=np.eye(3), shape=(2, 3), initval=np.zeros((2, 3)))
 
+    logp_fn = get_jaxified_logp(model)
+
     # No jitter
     ips = _get_batched_jittered_initial_points(
-        model=model, chains=1, random_seed=1, initvals=None, jitter=False
+        model=model, chains=1, random_seed=1, initvals=None, jitter=False, logp_fn=logp_fn
```

Comment:
We don't have to change these tests. Do you think we should?

Reply:
`_get_batched_jittered_initial_points` now takes a new parameter with no default so I had to supply it here

Reply:
You can make it accept `None`, in that case it will default to `model.compile_logp()` which is fine?

Reply:
Certainly possible to go that way, but given it's `test_jax.py`, maybe it's better to test closer to how it will actually be used? I could parametrize the test and do both?

Reply:
May be overkill. The point of this test was just to confirm we get different initvals with jitter. The logp function shouldn't play much of a role since we don't expect -inf logps here

Reply:
Ok no problem. With the changes to the function signature I can revert these changes to the test